src/services/config.ts

```diff
@@ -53,8 +53,9 @@ class Config {
   }
 
   getLocalConfig() {
+    const basePath = process.env.BASE_URL || './'
     return this.$http
-      .get('./config.json')
+      .get(`${basePath}config.json`)
```

Comment:
@vykulakov This assumes that a `BASE_URL` always has a trailing slash. This is the same assumption as for `index.html`.

Reply:
Looks good for me.